models/event-subEvent.json

```diff
@@ -2,6 +2,33 @@
   "title": "event.subEvent",
   "type": "array",
   "description": "A list of occurences of the [event](./event.json) when using the `single` or `multiple` [calendarType](./event-calendarType.json).",
+  "x-examples": {
```

Comment:
Should these examples also include a `bookingAvailability` property?

Reply:
Yes good that you spotted this! I've added them in 38b0b7b